tools/Dockerfile

```diff
@@ -1,19 +1,9 @@
-FROM ubuntu
-MAINTAINER zikalino
+FROM mcr.microsoft.com/dotnet/sdk:5.0.407-focal
+LABEL authors="zikalino,scbedd"
 
 RUN apt-get update
 RUN apt-get install -y git curl gnupg vim python3 python3-pip git software-properties-common apt-transport-https wget python3-venv nodejs npm libunwind-dev
 
-# install dotnet
```

Comment:
We can simply consume from a dotnet/sdk image instead of installing this entire thing for ourselves.